src/axom/mint/docs/sphinx/sections/tutorial.rst

```diff
@@ -0,0 +1,1255 @@
+.. ## Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _sections/tutorial:
+
+Tutorial
+---------
+
+The :ref:`sections/tutorial` section consists of simple examples and
+code snippets that illustrate how to use Mint's core classes and
+functions to construct and operate on the various supported :ref:`MeshTypes`.
+The examples presented herein aim to illustrate specific Mint concepts and
+capabilities in a structured and simple format.
+To quickly learn basic Mint concepts and capabilities through an illustrative
+walk-through of a complete working code example, see the
+:ref:`sections/getting_started` section. Additional code examples, based on
+Mint mini-apps, are provided in the :ref:`sections/examples` section. For
+thorough documentation of the interfaces of the various classes and functions
+in Mint, developers are advised to consult the
+`Mint Doxygen API Documentation`_, in conjunction with this
+:ref:`sections/tutorial`.
+
```

Comment:
It feels to me like there is too much repetition or duplication between (for example) mesh_types.rst and this "meshes" section of tutorial.rst.
- You can probably just remove the first paragraph from each of the sections of tutorial.rst: that would make the tutorial shorter, easier to understand, and thus more useful.
- You might also consider folding each section of the tutorial into the appropriate higher-level document so that (for example) execution_model.rst would both introduce the ideas and show examples of their use.  (I personally like the idea of folding examples in with concepts, but you're the one writing this document.)

Reply:
There is some overlap, but, that's intentional. I don't expect people to read this documentation sequentially. Some one may jump straight to the tutorial section. I wanted the tutorial to be somewhat self-contained with links back to the concepts to provide more information.

I did think of folding the tutorial with the concepts, but, I felt that it would have been more difficult to navigate and find where things are.